Environments/WebApp-PostgresSQL/Readme.md

```diff
@@ -0,0 +1,5 @@
+# Azure Web App with Postgres SQL
+
+Azure Web Apps enable you to build and host web applications in the programming language of your choice without managing infrastructure. It offers auto-scaling and high availability and enable automated deployments from GitHub,Visual Studio Team Services, or any Git repo. 
```

Comment:
Same here. It seems the text is copied. So, the comments elsewhere also apply throughout.